content/en/docs/started/support.md

```diff
@@ -4,224 +4,99 @@ description = "Where to get support for Kubeflow"
-
-The following organizations offer advice and support for Kubeflow deployments:
-
-<div class="table-responsive">
-  <table class="table table-bordered">
-    <thead class="thead-light">
-      <tr>
-        <th>Organization</th>
-        <th>Points of contact</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td><a href="https://www.arrikto.com">Arrikto</a></td>
-        <td><a href="mailto:sales@arrikto.com">sales@arrikto.com</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.ubuntu.com">Canonical</a></td>
-        <td><a href="https://ubuntu.com/kubeflow#get-in-touch">Get in touch</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.pattersonconsultingtn.com/">Patterson Consulting</a></td>
-        <td> 
-        <a href="http://www.pattersonconsultingtn.com/offerings/managed_kubeflow.html">Managed Kubeflow</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.seldon.io/">Seldon</a></td>
-        <td> 
-        <a href="https://github.com/SeldonIO/seldon-core/issues">Issue 
-        tracker</a></td>
-      </tr>    
-    </tbody>
-  </table>
-</div>
+## Support from commercial providers in the Kubeflow Ecosystem
 
-<a id="cloud-support"></a>
-## Support from a cloud or platform provider
+We want to promote commercial companies and idividuals that contribute back to the open source project.
+Below is a table of organizations that contribute to Kubeflow and offer commercial support:
 
-If you're using the services of a cloud provider to host Kubeflow, the cloud
-provider may be able to help you diagnose and solve a problem.
+| Provider                      | Support Link                                                                                                                                                    |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aranui Solutions              | [Aranui Solutions](https://www.aranui.solutions/services)                                                                                                       |
+| Canonical                     | [Ubuntu Kubeflow](https://ubuntu.com/kubeflow#get-in-touch)                                                                                                     |
+| Freelancer Julius von Kohout  | [LinkedIn](https://de.linkedin.com/in/juliusvonkohout/), [Slack](https://cloud-native.slack.com/team/U06LW431SJF), [GitHub](https://github.com/juliusvonkohout) |
+| Red Hat                       | [Red Hat](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai)                                                                        |
+| Other Providers               | Please reach out to the Kubeflow Steering Committee with proof of significant contributions to the Kubeflow open source project                                 |
```

Comment:
Can we get confirmation from @kubeflow/red-hat @thesuperzapper @mvlassis @StefanoFioravanzo @DnPlas @kimwnasptd about their commitment to commercial support please ?

Reply:
I mean that is the business model for redhat and canonical.

Reply:
I understand this, just want to get +1 from these folks to be listed in this page.

Reply:
@thesuperzapper Please can you give your lgtm if you are ok to be listed in this list.

Reply:
Done in https://github.com/kubeflow/website/pull/4021#issuecomment-2692932505

Reply:
Canonical was there before and Redhat agreed as well.